slsDetectorSoftware/src/Detector.cpp

```diff
@@ -405,6 +454,104 @@ Result<int> Detector::getTemperature(defs::dacIndex index,
     return res;
 }
 
+std::vector<defs::dacIndex> Detector::getDacList() const {
+    std::vector<defs::dacIndex> retval;
+    int i = 0;
+    switch (getDetectorType().squash()) {
+    case defs::EIGER:
```

Comment:
Same as for settings, we don't really need to build up the vector. 
Probably also workes with just return {dac, dac, dac, ...}

Reply:
done